### DISCUSSION TITLE
Spread in object literals

### DISCUSSION THREAD

**Discussion by Abdalla-Sleman** [dotnet/csharplang#7801] [Language Ideas] [answered]
Copying and merging objects : 

**example in javaScript you can do somthing like this :**

const obj1 = { foo: "bar", x: 42 };
const obj2 = { bar: "baz", y: 13 };

const mergedObj = { ...obj1, ...obj2 };
// { foo: "bar", x: 42, bar: "baz", y: 13 }

**Comment by colejohnson66** [comment]
How would this work in C# where types must be defined up front?

**Reply by KennethHoff** [reply]
It could be exclusively for anonymous objects, but I'm guessing that's not what he wants

**Comment by HaloFour** [comment, marked as answer]
See: #2879
